Extended/TIPXWebPCodec.m

```diff
@@ -15,12 +15,8 @@
 
 #pragma mark WebP includes
 
-#if TARGET_OS_MACCATALYST
-#import <webp/webp.h>
-#else
-#import <WebP/decode.h>
-#import <WebP/encode.h>
-#endif
+#import <libwebp/decode.h>
+#import <libwebp/encode.h>
```

Comment:
Maybe this needs to be like this: https://github.com/SDWebImage/SDWebImageWebPCoder/blob/84c97db988a0cc81cf419828bc9606a5a2b26a06/SDWebImageWebPCoder/Classes/SDImageWebPCoder.m#L11-L23

I'm not sure though... Need to dig a little